src/webcore/value.rs

```diff
@@ -1291,4 +1297,50 @@ mod tests {
         let a: Result< (), _ > = js! { return 1; }.try_into();
         assert!( a.is_err() );
     }
+
+    #[test]
+    fn reference_stable() {
+        js! { Module.__test = {}; }
+        let a = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        let b = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        assert_eq!(a, b);
+
+        js! { delete Module.__test; }
+        js! { Module.__test = {}; }
+        let c = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        assert_ne!(a, c);
+
+        js! { Module.__test = {}; }
+    }
+
+    fn assert_noref(refid: i32) {
+        let has_refcount: bool = js! {
+            return @{refid} in Module.STDWEB_PRIVATE.id_to_refcount_map;
+        }.try_into().unwrap();
+
+        assert!(!has_refcount);
+
+        let has_ref: bool = js! {
+            return @{refid} in Module.STDWEB_PRIVATE.id_to_ref_map;
+        }.try_into().unwrap();
+
+        assert!(!has_ref);
+    }
+
+    #[test]
+    fn reference_refcount() {
+        let refid = {
+            let obj = js! { return new Object(); };
+            obj.as_reference().unwrap().as_raw()
+        };
+        assert_noref(refid);
```

Comment:
Could you make this more explicit? e.g.:

```
let value = js! { return new Object(); }.into_reference().unwrap();
let refid = value.as_raw();
assert_eq!( is_known_reference(refid), true );
mem::drop( value );
assert_eq!( is_known_reference(refid), false );
```

Basically, to explicitly show that dropping the `Reference` will also remove it from the map.

Reply:
I changed the patch to use `drop` explicitly.